Our system-auth and common-auth are either not getting called through
the dependency chain or are not including pam_systemd.so to start a
user session.

We only want a user session when it's really needed, so add it right
in this file to avoid getting user sessions on other vectors.
--- a/src/login/systemd-user.m4.org	2016-07-22 10:33:25.974491417 -0700
+++ b/src/login/systemd-user.m4	2016-07-22 10:51:55.861867272 -0700
@@ -9,3 +9,4 @@
 session  required pam_selinux.so nottys open
 )m4_dnl
 session  include system-auth
+-session optional pam_systemd.so
